data-sources/amazon-cloudwatch-internet-monitor/config.yml

```diff
@@ -0,0 +1,14 @@
+id: amazon-cloudwatch-metric-streams
```

Comment:
We already have a data source for amazon-cloudwatch-metric-streams. If you'd like, you can us e that data source instead and delete this file. 

https://github.com/newrelic/newrelic-quickstarts/blob/main/data-sources/amazon-cloudwatch-metric-streams/config.yml

Reply:
Seems like you're already referencing the data source i linked above so let' delete this file.

Reply:
deleted